.github/workflows/cygwin-windows.yml

```diff
@@ -0,0 +1,47 @@
+# Based on https://github.com/mit-plv/fiat-crypto/blob/master/.github/workflows/coq-windows.yml
+name: CI (Cygwin/Windows)
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  build:
+
+    runs-on: windows-latest
+
+    env:
+      CFLAGS: "--coverage -O2 -g"
+      CXXFLAGS: "--coverage -O2 -g"
+      LDFLAGS: "--coverage"
+      # default config flags: enable debug asserts
+      CONFIGFLAGS: "--enable-debug"
+      COVERALLS_PARALLEL: true
+      TEST_SUITES: testinstall
+
+    steps:
+    # This sets git to use Linux line endings. While GAP should allow
+    # Windows line endings in GAP files, we currently can't build GAP if the
+    # whole source has Windows line endings
+    - name: Set git to use LF
+      run: |
+        git config --global core.autocrlf false
+        git config --global core.eol lf
```

Comment:
Is this still necessary, now that we have a `.gitattributes` file?

Reply:
I believe .gitattribute is about how things are stored in the repository, not how they are checked out.

Reply:
Removing that fails for me:
https://github.com/ssiccha/gap/runs/1925397861?check_suite_focus=true